packages/back-end/src/routers/tag/tag.controller.ts

```diff
@@ -77,6 +78,9 @@ export const deleteTag = async (
   // features
   await removeTagInFeature(org, id);
 
+  // Slack integrations
+  await removeTagFromSlackIntegration({ organizationId: org.id, tag: id });
```

Comment:
it feels a bit strange that all of these "removeXinY" methods have slightly different interfaces. Should we standardize them?

Reply:
Good question. I don't think it's necessary to make them all the same. 

I do think the options object is a better pattern in terms of readability, extensibility, ease of implementation, and avoiding human error, but we do not have anything formally in our style guide around that (e.g. preferred pattern vs. number of arguments, etc.).

I also don't think it makes sense to take the time to update all the existing methods just so they match, nor trying to abstract a common interface. In my opinion, a good time to consider updating them to use an options object argument is when we need to add another argument to them. To avoid the case where someone adds another positional argument to a method with existing positional arguments but doesn't change the signature to use an options object, I've opted for an options object from the start.

Let me know your thoughts.

Reply:
I agree that an options object makes the most sense. We don’t need to tackle updating all such methods at the moment, but at least we’re aware of the code smell